taichi/python/export_ggui.cpp

```diff
@@ -287,10 +288,19 @@ struct PyWindow {
     window = std::make_unique<vulkan::Window>(prog, config);
   }
 
+  py::tuple get_window_shape() {
+    auto [w, h] = window->get_window_shape();
```

Comment:
OOC: is there any difference between resolution saved in Appconfig and w&h saved in swap_chain? Looks like the former is easier to get?